### PR TITLE
Yuidoc 4 - down to render action just before watch

### PR DESCRIPTION
**Pull request by SteveMcArthur**
nearly there

### CODE REVIEW

File: src/lib/docpad.coffee [hunk 1]
```diff
@@ -3500,25 +3536,55 @@ class DocPad extends EventEmitterGrouped
...
-	# Add Model
-	addModel: (model, opts) ->
+	###*
+	# Add supplied model to the DocPad database
+	# @method addModel
+	# @param {Object} model
+	# @param {Object} opts
+	# @return {Object} the model
+	###
+		addModel: (model, opts) ->
```
Comment (balupton) [inline src/lib/docpad.coffee]: `addModel` has been accidentally indented

**Comment by balupton** [comment]
One issue, once fixed, good to merge!

**Comment by balupton** [comment]
:+1:

**Comment by pflannery** [comment]
@SteveMcArthur awesome work so far!

**Pull request merged**